webapp/javascript/components/TimelineChart/Selection.plugin.ts

```diff
@@ -346,6 +315,18 @@ const handleHeight = 22;
     plot.setSelection = setSelection;
     plot.getSelection = getSelection;
 
+    function onHoveringOnAnnotation(
+      _: EventType,
+      value?: { hovering: boolean }
+    ) {
+      // during further complication this logic should be moved to separated plugin
+      // which only handles cursor state over Flot
+      // because cursor setters conflict between each other
```

Comment:
Yeah not a fan of this at all,

If we implement icons as react elements this won't be a problem correct?

Reply:
removed